sdk/java-sdk-spring/src/test/scala/kalix/javasdk/impl/ActionDescriptorFactorySpec.scala

```diff
@@ -417,6 +420,22 @@ class ActionDescriptorFactorySpec extends AnyWordSpec with ComponentDescriptorSu
       }.getMessage should include("You cannot use @Subscribe.EventSourcedEntity annotation in both methods and class.")
     }
 
+    //TODO remove ignore after updating to Scala 2.13.11 (https://github.com/scala/scala/pull/10105)
+    "validates if there are missing event handlers for event sourced Entity Subscription at type level" ignore {
+      intercept[InvalidComponentException] {
+        Validations.validate(classOf[MissingHandlersWhenSubscribeToEventSourcedEntityAction]).failIfInvalid
+      }.getMessage should include(
+        "Component `MissingHandlersWhenSubscribeToEventSourcedEntityAction` is missing event handler for `kalix.spring.testmodels.eventsourcedentity.EmployeeEvent$EmployeeEmailUpdated`")
```

Comment:
Backticks are being used while the message will have single-quotes

Reply:
fixed